### PR TITLE
Removed the sub-titles for sym and asym encryption

### PR DESCRIPTION
**Pull request by himeshsiriwardana**
screencapture-localhost-8000-en-next-deploy-security-encrypt-passwords-with-cipher-tool-2024-10-31-16_03_59

### CODE REVIEW

File: en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+{% if product_name == "WSO2 Identity Server" and is_version != "7.0..0" %}
+## Rotating Encryption Secrets
+
+!!! note
+    To support symmetric encryption, you must specify a PKCS12 type keystore as the internal keystore.
+
+You can rotate encryption keys by switching between symmetric and asymmetric encryption or by changing the encryption keys within the same encryption mode.
+
+1. For **symmetric** encryption, use the command below to add the new key to the existing keystore with a new alias.
```
Comment (hwupathum) [inline en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md]: Shall we use the === "Symmetric encryption" sections here too, for consistency?

File: en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+3. Open the `deployment.toml` file again and see that the alias passwords are encrypted.
+
+    ```toml
+    [secrets]
+    admin_password = "GeNld2aZkydnIZGtkZYOnXlVzl8WBtZzAQ8kIoR5c7aHmyUkWTag7w4dG6B3JK5GxeX9bhsmZCBFozlPdWBT6Jvy"
+    keystore_password = "brClL1SOHdezXTvBz1/76b/DnHQgxjNGtzhaBr3DnhHw32NWY484abHLREVyMoNJkER5lQUPbqeaMpR5lQUPbqeaMp"
+    key_password = "CFAaISaI19dHLApEM3usNSDXXdhdicHbVncrVwuLDJp6Rhp8B3Qy3PnBhcJsryTqR/EPwdLnXboNJkER"
+    truststrore_password = "DKnecEw+mJ8JhTUrqxpTZxwXrOdtcoAl2hD3LHtH+yJXNogumdSALfaqrMaknBzJq4SF3sY0RvwkMxWhnZ+BhIsko"
+    "log4j.appender.LOGEVENT.password" = "kydnIZGtkZYOnXlVzl8WBtZzAQ8kIoR5c7aHmyUkWTagXTvBz1/76b/DnHQgxjNhD3LHtH+yJXNowecEEC"
```
Comment (hwupathum) [inline en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md]:
```suggestion
    log4j2_password = "xxxxxxx"
```

File: en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+3. Open the `deployment.toml` again and see that the passwords are encrypted.
+
+    ```toml
+    [secrets]
+    admin_password = "xxxxxxx"
+    keystore_password = "xxxxxxx"
+    key_password = "xxxxxxx"
+    truststrore_password = "xxxxxxx"
+    "log4j.appender.LOGEVENT.password" = "xxxxxxx"
```
Comment (hwupathum) [inline en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md]:
```suggestion
    log4j2_password = "xxxxxxx"
```

File: en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+2. Open a terminal, navigate to the `<IS_HOME>/bin/` directory, and execute the following command to enable the cipher tool using the `-Dconfigure` flag.
+
+    - On Linux: `./ciphertool.sh -Dconfigure`
+    - On Windows: `ciphertool.bat -Dconfigure`
+
+3. Open the `deployment.toml` file again and see that the alias passwords are encrypted.
+
+    ```toml
+    [secrets]
+    admin_password = "GeNld2aZkydnIZGtkZYOnXlVzl8WBtZzAQ8kIoR5c7aHmyUkWTag7w4dG6B3JK5GxeX9bhsmZCBFozlPdWBT6Jvy"
```
Comment (hwupathum) [inline en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md]: We can use the "xxxxx..." format here too

Comment (himeshsiriwardana) [reply en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md]: fixed with https://github.com/hwupathum/docs-is/pull/5/commits/3df8725bb45a9cfc4c10b1ae1c1237de5715212f

File: en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md (hunk 3 above)
Comment (himeshsiriwardana) [reply en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md]: fixed with https://github.com/hwupathum/docs-is/pull/5/commits/3df8725bb45a9cfc4c10b1ae1c1237de5715212f

File: en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md (hunk 2 above)
Comment (himeshsiriwardana) [reply en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md]: fixed with https://github.com/hwupathum/docs-is/pull/5/commits/3df8725bb45a9cfc4c10b1ae1c1237de5715212f

File: en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md (hunk 1 above)
Comment (himeshsiriwardana) [reply en/includes/deploy/security/encrypt-passwords-with-cipher-tool.md]: fixed with https://github.com/hwupathum/docs-is/pull/5/commits/3df8725bb45a9cfc4c10b1ae1c1237de5715212f

**Review by hwupathum — APPROVED** [review]

**Pull request merged**